### PR TITLE
Improve content store handling

### PR DESCRIPTION
**Pull request by teneightfive**
This includes a refactor to allow the content store library to
support multiple handlers.

### CODE REVIEW

File: lib/content-store/file-handler.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+const path = require('path');
+const fs = require('fs');
+
+const BASEFILEPATH = path.resolve(__dirname, '..', '..', 'content');
```
Comment (tyom) [inline lib/content-store/file-handler.js]: `BASE_FILE_PATH` would probably read better.

File: lib/content-store/file-handler.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+function loadFile(filepath) {
+  let filecontents;
+
+  try {
+    fs.accessSync(filepath, fs.F_OK);
+    filecontents = fs.readFileSync(filepath, 'utf8').trim();
+  } catch (e) {
+    filecontents = false;
```
Comment (tyom) [inline lib/content-store/file-handler.js]: Should it be `null`? The other value is a string so it makes sense to present the absence of that value as `null`.

File: lib/content-store/file-handler.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+const path = require('path');
+const fs = require('fs');
+
+const BASEFILEPATH = path.resolve(__dirname, '..', '..', 'content');
+
+function loadFile(filepath) {
+  let filecontents;
```
Comment (tyom) [inline lib/content-store/file-handler.js]: camelCase?

File: lib/content-store/file-handler.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    filecontents = fs.readFileSync(filepath, 'utf8').trim();
+  } catch (e) {
+    filecontents = false;
+  }
+
+  return filecontents;
+}
+
+function transformData(obj, dirpath) {
+  return Object.keys(obj).reduce((previous, key) => {
```
Comment (tyom) [inline lib/content-store/file-handler.js]: Should we ensure that `obj` is indeed an object? e.g. `lodash.isObject`.

File: lib/content-store/file-handler.js (hunk 2 above)
Comment (teneightfive) [inline lib/content-store/file-handler.js]: Yeah, good point. Will change it to `null`

**Pull request merged**